docs/rooms/fireRoom.md

```diff
@@ -0,0 +1,59 @@
+# Fire Room
+
+A room with three different paths.
+One path leads to a person, the second path leads to a item and the third path leads no where.
+Each path is on fire but the player is given two buckets of water.
+The goal of this room is to save the person.
+
+## PlayerStates
```

Comment:
Can you also add a "RoomState" section to cover the shape of the  room data? You can use [example room 3](https://github.com/normal-devs/klicker-knight/blob/b2e0c64bf9ab5cf02c852806e7528ec75e13b8a0/docs/rooms/exampleRoom3Room.md?plain=1#L15-L17)  as an example